### ISSUE TITLE
Can't sign into rumble via Google button

### ISSUE THREAD

**Issue by Cn116** [futo-org/grayjay-android#1206]
### What happened?

It's works on rumble but doesn't not work on grayjay. The loading symbols does the same thing endlessly. 
![Screenshot_20240731-135704](https://github.com/user-attachments/assets/cea385d9-d6e2-46c5-be9f-ec26e715bca4)

### Grayjay Version

253

### What plugins are you seeing the problem on?

Rumble

### Plugin Version

44

### When do you experience the issue?

- [ ] While logged in
- [ ] While logged out
- [X] N/A

### Relevant log output

```shell
How do I enable logging????
```

**Comment by mooneyto** [comment]
I am also having a similar issue. When I try to login with Google it says "This browser or app may not be secure." I've never been able to login to rumble since launch. Kind of sucks, not much of a point to the plugin if it doesn't work.

**Comment by Cn116** [comment]
You are right like I shouldn't have to delete this account and make one with email just so it works.

**Comment by codebymatter** [comment]
I am signed in to Rumble on Grayjay. I have seen thst warnung message and honestly no issues. I had to sign back in sometimesfir engaging with Live Chats abd not have issues. Are both of you on latesf Abdroid OS, by chance?

**Comment by mooneyto** [comment]
Yeah, updated os and apps. No updates available. It has never let me login to rumble. Before this error it just sat on a blank screen.

**Comment by codebymatter** [comment]
I wonder if some bugs relafed to Grayjay or plugins sre related to Android 15. I'm on Amdroid 12 by the way.

**Comment by Cn116** [comment]
I am on the latest is and have the latest July security patch. I am currently using a pixel 6 pro and grayjay is updated.

**Comment by mooneyto** [comment]
This is still a problem? Now that rumble has premium only content this is even more of an issue. Would be great to be able to log in.
